docs/lib/datastore/sync.md

```diff
@@ -3,8 +3,97 @@ title: Syncing data to cloud
 description: Learn more about how DataStore connects to an AppSync backend and automatically syncs all locally saved data using GraphQL.
 ---
 
-Once you're happy with your application, you can start syncing with the cloud by provisioning a backend from your project. DataStore can connect to an AppSync backend and automatically sync all locally saved data using GraphQL as a data protocol.
+Once you're happy with your application, you can start syncing with the cloud by provisioning a backend from your project. DataStore can connect to remote backend and automatically sync all locally saved data using GraphQL as a data protocol.
 
-<inline-fragment platform="js" src="~/lib/datastore/fragments/js/sync.md"></inline-fragment>
-<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/sync.md"></inline-fragment>
-<inline-fragment platform="android" src="~/lib/datastore/fragments/android/sync.md"></inline-fragment>
\ No newline at end of file
+<amplify-callout>
+
+**Best practice:** it is recommended to develop without cloud synchronization enabled initially so you can change the schema as your application takes shape without the impact of having to update the provisioned backend. Once you are satisfied with the stability of your data schema, setup cloud synchonization as described below and the data saved locally will be synchornized to the cloud automatically.
+
+</amplify-callout>
+
+## Setup cloud sync
+
+Synchronization between offline and online data can be tricky. DataStore goal is to remove that burden from the application code and handle all data consistency and reconciliation between local and remote behind the scenes, while developers focus on their application logic. Up to this point the focus was to setup a local datastore that works offline and has all the capabilities you would expect from a data persistence framework.
```

Comment:
"up to this point" is ambiguous.... do we mean.. prior to launching amplify? or... something else?

Reply:
No, the intent was to say up to that point in the docs (if you go page by page). A more clear sentence would definitely be better